pkgs/applications/science/programming/fdr/default.nix

```diff
@@ -0,0 +1,87 @@
+{stdenv, fetchurl, qt57, ncurses, xorg, zlib, python27Packages}:
+let libs = [
+  stdenv.cc.cc
+  python27Packages.python
+  qt57.qtbase
+  qt57.qtx11extras
+  ncurses
+  xorg.libX11
+  xorg.libXft
+  zlib
+  ];
+in
+stdenv.mkDerivation {
+  name = "fdr";
+  src = fetchurl {
+    url = https://www.cs.ox.ac.uk/projects/fdr/downloads/fdr-3754-linux-x86_64.tar.gz;
+    md5 = "c09a05a5d9b54fc65ed12de8b7cdfc11";
```

Comment:
We no longer allow packages to use md5 for verification purposes, please use the sha256 (which can be found via `nix-prefetch-url https://www.cs.ox.ac.uk/projects/fdr/downloads/fdr-3754-linux-x86_64.tar.gz`.